src/encoded/schemas/micro_rna_quantification_quality_metric.json

```diff
@@ -0,0 +1,41 @@
+{
+    "title": "microRNA quantification quality metric",
+    "description": "Schema for reporting microRNA pipeline quantification quality metric",
+    "id": "/profiles/micro_rna_quantification_quality_metric.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "required": ["step_run", "award", "lab", "quality_metric_of", "expressed_mirnas"],
+    "additionalProperties": false,
+    "identifyingProperties": ["uuid"],
+    "mixinProperties": [
+        { "$ref": "mixins.json#/schema_version" },
+        { "$ref": "quality_metric.json#/properties" },
+        { "$ref": "mixins.json#/uuid" },
+        { "$ref": "mixins.json#/submitted"},
+        { "$ref": "mixins.json#/aliases"},
+        { "$ref": "mixins.json#/attribution" },
+        { "$ref": "mixins.json#/standard_status"},
+        { "$ref": "mixins.json#/assay" }
+   ],
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "expressed_mirnas": {
+            "title": "Expressed miRNAs",
+            "description": "Number of miRNAs expressed",
+            "comment": "miRNAs expressed < 2 cpms are filtered",
```

Comment:
If this comment is needed, I think come clarification may be good. "miRNAs expressed < 2 cpms are not counted as expressed" or something like that.